tests/python/pants_test/build_graph/BUILD

```diff
@@ -1,23 +1,9 @@
 # Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_tests(
-  name = 'address',
-  sources = ['test_address.py'],
-)
-
-python_tests(
-  name = 'build_configuration',
-  sources = ['test_build_configuration.py'],
-)
+python_tests()
```

Comment:
What is preventing this from picking up the same tests as `python_integration_tests` ? Is the glob there disjoint from the default for `python_tests`?

Reply:
Ah, good catch. This is indeed catching integration tests currently.